HousingRepairsOnlineApi/Factories/ResponseFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using HousingRepairsOnlineApi.UseCases;
+using Notify.Models.Responses;
+
+namespace HousingRepairsOnlineApi.Factories
+{
+    public static class ResponseFactory
```

Comment:
I would say this isn't quite a "factory" - it converts from one type to another.

I think it would be better suited in an XXXExtensions class within the `Domain` namespace, i.e. `HousingRepairsOnlineApi.Domain.SmsNotificationResponseExtensions`.